Jamfile

```diff
@@ -7,16 +7,4 @@
 # Official repository: https://github.com/CPPAlliance/buffers
 #
 
-import ../../config/checks/config : requires ;
-
-constant c11-requires :
-    [ requires
-    cxx11_constexpr
-    cxx11_decltype
-    cxx11_hdr_tuple
-    cxx11_template_aliases
-    cxx11_variadic_templates
-    ]
-    ;
-
 build-project test ;
```

Comment:
Is this necessary?

Reply:
That's actually a good question. I can try digging into this to see if this is genuinely necessary or not.

Reply:
I believe the point of that is so "b2" from the root will build tests. But I am not 100% sure.

Reply:
Ah, so it is. I've tested locally. Without it, no tests run from the project root. Running `b2 libs/buffers` still works as expected. We can keep this. I don't mind now that we know the functionality.

Do you guys think I should update the root Jamfile to contain a comment summarizing this?

Reply:
> Without it, no tests run from the project root.

Do we need to be able to run tests from the root?

Reply:
Peter doesn't like a top-level Jamfile so we should get rid of it.